changelog/fragments/1714505172-Send-fleet-server-component-output-under-bootstrap-key.yaml

```diff
@@ -0,0 +1,39 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: Send fleet-server component output under bootstrap key
+
+# Long description; in case the summary is not enough to describe the change
+# this field accommodate a description without length limits.
+# NOTE: This field will be rendered only for breaking-change and known-issue kinds at the moment.
+description: |
+  Alter the fleet-server bootstrap component modifier to insert all (elasticsearch)
+  output configuration options specified by enrollment args under a new elasticsearch.boostrap key
+  instead of overwriting any existing keys. This will allow elastic-agent to send the list of hosts
+  (and other config options) retrieved from a policy to fleet server as well as the config needed
+  to form the initial connection to elasticsearch used to collect policy information.
+  Fleet-server will be altered to use the bootstrap config that is passed if the policy attributes
+  are unspecified or fail.
```

Comment:
Be less technical in this. This is what our users will read when wanting to upgrade, so tell them the benefits.